src/app/containers/Image/index.jsx

```diff
@@ -33,7 +44,7 @@ const ImageContainer = ({ blocks }) => {
     <Figure
       src={rawImageSrc}
       alt={altText}
-      copyrightHolder={copyrightHolder}
+      copyright={getCopyright(copyrightHolder)}
```

Comment:
Can you move `getCopyright` and `getCaption` out of being executed here, and define it as a variable above, like for `rawImageSrc` and `altText`? For brevity.